crates/python/src/lib.rs

```diff
@@ -68,3 +68,10 @@ fn reset_logging() {
         }
     }
 }
+
+py_sync::py_function_sync_async! {
+    #[cfg()]
```

Comment:
This was to satisfy the necessary `meta` in the macro pattern, which - if possible - we may want to make optional

Reply:
The build failures were due to the missing `#[pyfunction]` attribute, so adding it fixes those and satisfies the macro pattern. I believe the reason it's not included in the macro by default, is because it makes it hard to customize the attribute, which we commonly do (e.g. `#[pyfunction(client = "None", options = "None")]).

Reply:
aha - thanks. Worth adding to the macro docstring, or is it omitted for a reason?

Reply:
I can't think of a good reason, probably an over sight, added it!